core/dbt/clients/jinja.py

```diff
@@ -428,7 +434,11 @@ def get_environment(
     else:
         env_cls = MacroFuzzEnvironment
 
-    return env_cls(**args)
+    env = env_cls(**args)
+    if native:
+        env.filters['as_text'] = TextMarker
```

Comment:
can we always add this?

Reply:
Sure! I think in the else case we actually want to set it to `str`.